packages/pigment-css-core/src/processors/css.ts

```diff
@@ -232,21 +249,44 @@ export class CssObjectProcessor extends BaseCssProcessor {
 
   getDependencies(): ExpressionValue[] {
     const [, ...params] = this.callParam;
-    return params.flat().filter((param) => 'kind' in param);
+    return params.flat().filter((param) => 'kind' in param && param.kind !== ValueType.CONST);
+  }
+
+  isMaybeTransformedTemplateLiteral(values: ValueCache): boolean {
+    const [, firstArg, ...restArgs] = this.callParam;
+    if (!('kind' in firstArg) || firstArg.kind === ValueType.CONST) {
```

Comment:
I don't think I've used the `in` operator in over a decade 😄
It's purely stylistic in this instance, `.hasOwnProperty` feels more predictable to me.

Reply:
I think `in` makes TS happy too.

Reply:
Correct. `in` narrows the types to the expected ones.